pkg/kubelet/eviction/helpers_test.go

```diff
@@ -32,7 +32,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	statsapi "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
-
```

Comment:
can revert changes to this file completely

Reply:
Done changes